akeyless_gateway/metadata.csv

```diff
@@ -0,0 +1,9 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+akeyless.gw.system.cpu,gauge,,,,CPU utilization metrics,0,akeyless_gateway,,
+akeyless.gw.system.disk,gauge,,,,Disk I/O metrics,0,akeyless_gateway,,
+akeyless.gw.system.load,gauge,,,,CPU load metrics,0,akeyless_gateway,,
+akeyless.gw.system.memory,gauge,,,,Memory utilization metrics,0,akeyless_gateway,,
+akeyless.gw.system.network,gauge,,,,Network interface I/O metrics & TCP connection metrics,0,akeyless_gateway,,
+akeyless_gw_quota_current_transactions_number,gauge,,,,Current transaction number,0,akeyless_gateway,,
+akeyless_gw_quota_gw_admin_client_transactions,gauge,,,,Total transaction by an admin client,0,akeyless_gateway,,
+akeyless_gw_quota_total_transactions_limit,gauge,,,,Total transaction limit per hour,0,akeyless_gateway,,
```

Comment:
```suggestion
akeyless.gw_quota_current_transactions_number,gauge,,,,Current transaction number,0,akeyless_gateway,,
akeyless.gw_quota_gw_admin_client_transactions,gauge,,,,Total transaction by an admin client,0,akeyless_gateway,,
akeyless.gw_quota_total_transactions_limit,gauge,,,,Total transaction limit per hour,0,akeyless_gateway,,
```

Can these be fixed to be in the the correct `namespace.` format?

Reply:
I may have committed that too quickly. I need to check with our team if this can be done.

Reply:
Ok - I was told we can't change that internally at this time. I reverted the commit. Is there an issue with the PR because of this?